### ISSUE TITLE
trivy status is unhealthy

### ISSUE THREAD

**Issue by jcambronero** [goharbor/harbor-helm#594]
Hi, I just deployed the latest version of the helm chart (v1.4.0) and I'm seeing that the trivy scanner status is set to unhealthy.
This is the error that I found in the log:
```
2020-06-01T19:26:44Z [ERROR] [/server/error/error.go:53]: {"errors":[{"code":"INTERNAL_SERVER_ERROR","message":"scanner API: get metadata: scanner controller: ping: v1 client: get metadata: general response handler: invalid character '\u003c' looking for beginning of value"}]}
```

These are all the trivy logs (with debug mode turned on):
```
ls: /harbor_cust_cert: No such file or directory
{"built_at":"2020-04-15T15:29:29Z","commit":"473a52406503e172472635b59245f9f573fdaa65","level":"info","msg":"Starting harbor-scanner-trivy","time":"2020-06-01T19:09:06Z","version":"0.9.0"}
{"level":"debug","msg":"Current process","pid":10,"time":"2020-06-01T19:09:06Z"}
{"gid":10000,"home_dir":"/home/scanner","level":"debug","msg":"Current user","time":"2020-06-01T19:09:06Z","uid":10000}
{"level":"warning","msg":"trivy cache dir does not exist","path":"/home/scanner/.cache/trivy","time":"2020-06-01T19:09:06Z"}
{"level":"debug","msg":"Creating trivy cache dir","path":"/home/scanner/.cache/trivy","time":"2020-06-01T19:09:06Z"}
{"level":"debug","mode":"dgrwxr-xr-x","msg":"trivy cache dir permissions","time":"2020-06-01T19:09:06Z"}
{"level":"warning","msg":"trivy reports dir does not exist","path":"/home/scanner/.cache/reports","time":"2020-06-01T19:09:06Z"}
{"level":"debug","msg":"Creating trivy reports dir","path":"/home/scanner/.cache/reports","time":"2020-06-01T19:09:06Z"}
{"level":"debug","mode":"dgrwxr-xr-x","msg":"trivy reports dir permissions","time":"2020-06-01T19:09:06Z"}
{"addr":":8080","level":"warning","msg":"Starting API server without TLS","time":"2020-06-01T19:09:06Z"}
```

Here's all the logs from harbor-core (debug mode also turned on):
```
ls: /harbor_cust_cert: No such file or directory
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.oci.image.index.v1+json registered
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.docker.distribution.manifest.list.v2+json registered
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.docker.distribution.manifest.v1+prettyjws registered
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.oci.image.config.v1+json registered
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.docker.container.image.v1+json registered
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.cncf.helm.config.v1+json registered
2020-06-01T19:25:04Z [INFO] [/controller/artifact/processor/processor.go:58]: the processor to process media type application/vnd.cnab.manifest.v1 registered
2020-06-01T19:25:04Z [DEBUG] [/pkg/permission/evaluator/rbac/casbin_match.go:65]: Starting regexp store purge in 39m0s
2020-06-01T19:25:04Z [INFO] [/replication/adapter/native/adapter.go:36]: the factory for adapter docker-registry registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/harbor/adaper.go:31]: the factory for adapter harbor registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/dockerhub/adapter.go:25]: Factory for adapter docker-hub registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/huawei/huawei_adapter.go:27]: the factory of Huawei adapter was registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/googlegcr/adapter.go:29]: the factory for adapter google-gcr registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/awsecr/adapter.go:47]: the factory for adapter aws-ecr registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/azurecr/adapter.go:15]: Factory for adapter azure-acr registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/aliacr/adapter.go:31]: the factory for adapter ali-acr registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/jfrog/adapter.go:30]: the factory of jfrog artifactory adapter was registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/quayio/adapter.go:38]: the factory of Quay.io adapter was registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/helmhub/adapter.go:30]: the factory for adapter helm-hub registered
2020-06-01T19:25:04Z [INFO] [/replication/adapter/gitlab/adapter.go:17]: the factory for adapter gitlab registered
2020-06-01T19:25:04Z [DEBUG] [/core/auth/authenticator.go:126]: Registered authentication helper for auth mode: http_auth
2020-06-01T19:25:04Z [DEBUG] [/core/auth/authenticator.go:126]: Registered authentication helper for auth mode: db_auth
2020-06-01T19:25:04Z [DEBUG] [/core/auth/authenticator.go:126]: Registered authentication helper for auth mode: ldap_auth
2020-06-01T19:25:04Z [DEBUG] [/core/auth/authenticator.go:126]: Registered authentication helper for auth mode: oidc_auth
2020-06-01T19:25:04Z [DEBUG] [/core/auth/authenticator.go:126]: Registered authentication helper for auth mode: uaa_auth
2020-06-01T19:25:04Z [DEBUG] [/pkg/notifier/topic/topics.go:23]: topic http is subscribed
2020-06-01T19:25:04Z [DEBUG] [/pkg/notifier/topic/topics.go:23]: topic slack is subscribed
2020-06-01T19:25:04Z [INFO] [/core/controllers/base.go:299]: Config path: /etc/core/app.conf
2020-06-01T19:25:04Z [INFO] [/core/main.go:111]: initializing configurations...
2020-06-01T19:25:04Z [INFO] [/core/config/config.go:83]: key path: /etc/core/key
2020-06-01T19:25:04Z [INFO] [/core/config/config.go:60]: init secret store
2020-06-01T19:25:04Z [INFO] [/core/config/config.go:63]: init project manager
2020-06-01T19:25:04Z [INFO] [/core/config/config.go:95]: initializing the project manager based on local database...
2020-06-01T19:25:04Z [INFO] [/core/main.go:113]: configurations initialization completed
2020-06-01T19:25:04Z [INFO] [/common/dao/base.go:84]: Registering database: type-PostgreSQL host-harbor-harbor-database port-5432 databse-registry sslmode-"disable"
2020-06-01T19:25:33Z [INFO] [/common/dao/base.go:89]: Register database completed
2020-06-01T19:25:33Z [DEBUG] [/migration/migration.go:41]: current database schema version: 0
2020-06-01T19:25:33Z [INFO] [/common/dao/pgsql.go:118]: Upgrading schema for pgsql ...
2020-06-01T19:25:34Z [DEBUG] [/migration/migration.go:57]: current data version: 0
2020-06-01T19:25:34Z [INFO] [/core/main.go:78]: User id: 1 updated its encrypted password successfully.
2020-06-01T19:25:34Z [INFO] [/chartserver/cache.go:184]: Enable redis cache for chart caching
2020-06-01T19:25:34Z [INFO] [/chartserver/reverse_proxy.go:60]: New chart server traffic proxy with middlewares
2020-06-01T19:25:34Z [DEBUG] [/core/api/chart_repository.go:612]: Chart storage server is set to http://harbor-harbor-chartmuseum
2020-06-01T19:25:34Z [INFO] [/core/api/chart_repository.go:613]: API controller for chart repository server is successfully initialized
2020-06-01T19:25:34Z [INFO] [/core/main.go:189]: Registering Trivy scanner
2020-06-01T19:25:34Z [INFO] [/common/dao/base.go:64]: initialized clair database
2020-06-01T19:25:34Z [INFO] [/core/main.go:211]: Registering Clair scanner
2020-06-01T19:25:34Z [INFO] [/pkg/scan/init.go:54]: Successfully registered Trivy scanner at http://harbor-harbor-trivy:8080
2020-06-01T19:25:34Z [INFO] [/pkg/scan/init.go:54]: Successfully registered Clair scanner at http://harbor-harbor-clair:8080
2020-06-01T19:25:34Z [INFO] [/core/main.go:229]: Setting Trivy as default scanner
2020-06-01T19:25:34Z [DEBUG] [/replication/replication.go:92]: the replication initialization completed
2020-06-01T19:25:34Z [INFO] [/core/main.go:156]: initializing notification...
2020-06-01T19:25:34Z [INFO] [/pkg/notification/notification.go:47]: notification initialization completed
2020-06-01T19:25:34Z [INFO] [/core/main.go:175]: Version: v2.0.0, Git commit: 87602132
2020/06/01 19:25:34.316 [I] [asm_amd64.s:1357] http server Running on http://:8080
2020-06-01T19:25:37Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 0a801cdf-6aa2-46ff-8dd5-d6ae753dad8e to the logger for the request GET /api/v2.0/ping
2020-06-01T19:25:37Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="0a801cdf-6aa2-46ff-8dd5-d6ae753dad8e"]: an unauthorized security context generated for request GET /api/v2.0/ping
2020/06/01 19:25:37.376 [D] [transaction.go:62] | 10.82.47.20| 200 | 303.678µs| match| GET /api/v2.0/ping r:/api/v2.0/ping
2020-06-01T19:25:45Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id c81c11ba-2e56-49f7-a294-c2533bc06ac4 to the logger for the request GET /api/internal/configurations
2020-06-01T19:25:45Z [DEBUG] [/server/middleware/security/secret.go:35][requestID="c81c11ba-2e56-49f7-a294-c2533bc06ac4"]: a secret security context generated for request GET /api/internal/configurations
2020/06/01 19:25:45.892 [D] [middleware.go:52] | 10.42.3.148| 200 | 1.801839ms| match| GET /api/internal/configurations r:/api/internal/configurations
2020-06-01T19:26:36Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 63d67a0b53e073878d8989ee11a7e6c3 to the logger for the request GET /api/v2.0/systeminfo
2020-06-01T19:26:36Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="63d67a0b53e073878d8989ee11a7e6c3"]: an unauthorized security context generated for request GET /api/v2.0/systeminfo
2020/06/01 19:26:37.005 [D] [transaction.go:62] | 10.254.207.28| 200 | 6.047367ms| match| GET /api/v2.0/systeminfo r:/api/v2.0/systeminfo
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id ea21d096f5e3e360ff0fad3a2be41af2 to the logger for the request GET /api/v2.0/users/current
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="ea21d096f5e3e360ff0fad3a2be41af2"]: an unauthorized security context generated for request GET /api/v2.0/users/current
2020-06-01T19:26:37Z [DEBUG] [/server/error/error.go:58]: {"errors":[{"code":"UNAUTHORIZED","message":"UnAuthorize"}]}
2020/06/01 19:26:37.033 [D] [transaction.go:62] | 10.254.207.28| 401 | 5.964436ms| match| GET /api/v2.0/users/current r:/api/v2.0/users/:id
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 01d180b472f2b6d7598ea86183185b34 to the logger for the request GET /api/v2.0/users/current
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="01d180b472f2b6d7598ea86183185b34"]: an unauthorized security context generated for request GET /api/v2.0/users/current
2020-06-01T19:26:37Z [DEBUG] [/server/error/error.go:58]: {"errors":[{"code":"UNAUTHORIZED","message":"UnAuthorize"}]}
2020/06/01 19:26:37.236 [D] [transaction.go:62] | 10.254.207.28| 401 | 686.226µs| match| GET /api/v2.0/users/current r:/api/v2.0/users/:id
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id ef9f7ee6-4241-4b44-892f-4627800a08b4 to the logger for the request GET /api/v2.0/ping
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="ef9f7ee6-4241-4b44-892f-4627800a08b4"]: an unauthorized security context generated for request GET /api/v2.0/ping
2020/06/01 19:26:37.375 [D] [transaction.go:62] | 10.82.47.20| 200 | 290.221µs| match| GET /api/v2.0/ping r:/api/v2.0/ping
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id b24dfd1fb573403a824a7109302deb14 to the logger for the request GET /api/v2.0/systeminfo
2020-06-01T19:26:37Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="b24dfd1fb573403a824a7109302deb14"]: an unauthorized security context generated for request GET /api/v2.0/systeminfo
2020/06/01 19:26:37.480 [D] [transaction.go:62] | 10.254.207.28| 200 | 819.755µs| match| GET /api/v2.0/systeminfo r:/api/v2.0/systeminfo
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 6d745ec4fd1de4d816a760ff2609e672 to the logger for the request POST /c/login
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="6d745ec4fd1de4d816a760ff2609e672"]: an unauthorized security context generated for request POST /c/login
2020-06-01T19:26:43Z [DEBUG] [/common/dao/user.go:277]: Check if user admin is super user
2020-06-01T19:26:43Z [DEBUG] [/core/auth/authenticator.go:139]: Current AUTH_MODE is db_auth
2020/06/01 19:26:43.130 [D] [middleware.go:52] | 10.254.207.28| 200 | 13.110581ms| match| POST /c/login r:/c/login
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 3ed04c6135c619b8b19d2283ebe66eb1 to the logger for the request GET /api/v2.0/users/current
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/security/session.go:47][requestID="3ed04c6135c619b8b19d2283ebe66eb1"]: a session security context generated for request GET /api/v2.0/users/current
2020/06/01 19:26:43.325 [D] [transaction.go:62] | 10.254.207.28| 200 | 1.895599ms| match| GET /api/v2.0/users/current r:/api/v2.0/users/:id
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id f35aa9c861c1805a68e2d1a9d17fe8c4 to the logger for the request GET /api/v2.0/scanners
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/security/session.go:47][requestID="f35aa9c861c1805a68e2d1a9d17fe8c4"]: a session security context generated for request GET /api/v2.0/scanners
2020/06/01 19:26:43.616 [D] [transaction.go:62] | 10.254.207.28| 200 | 2.54637ms| match| GET /api/v2.0/scanners r:/api/v2.0/scanners
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 9c362800c0aa23b247983079811261da to the logger for the request GET /api/v2.0/scanners
2020-06-01T19:26:43Z [DEBUG] [/server/middleware/security/session.go:47][requestID="9c362800c0aa23b247983079811261da"]: a session security context generated for request GET /api/v2.0/scanners
2020/06/01 19:26:43.807 [D] [transaction.go:62] | 10.254.207.28| 200 | 886.391µs| match| GET /api/v2.0/scanners r:/api/v2.0/scanners
2020-06-01T19:26:44Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 544e5ce6d00c610aa360af599f792322 to the logger for the request GET /api/v2.0/scanners/aa21ea03-a43d-11ea-b02e-42e080c4d65b/metadata
2020-06-01T19:26:44Z [DEBUG] [/server/middleware/security/session.go:47][requestID="544e5ce6d00c610aa360af599f792322"]: a session security context generated for request GET /api/v2.0/scanners/aa21ea03-a43d-11ea-b02e-42e080c4d65b/metadata
2020/06/01 19:26:44.013 [D] [transaction.go:62] | 10.254.207.28| 200 | 7.899113ms| match| GET /api/v2.0/scanners/aa21ea03-a43d-11ea-b02e-42e080c4d65b/metadata r:/api/v2.0/scanners/:uuid/metadata
2020-06-01T19:26:44Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 5a2da1b8c1d906dcaf409ff5b2d6983a to the logger for the request GET /api/v2.0/scanners/aa21c639-a43d-11ea-b02e-42e080c4d65b/metadata
2020-06-01T19:26:44Z [DEBUG] [/server/middleware/security/session.go:47][requestID="5a2da1b8c1d906dcaf409ff5b2d6983a"]: a session security context generated for request GET /api/v2.0/scanners/aa21c639-a43d-11ea-b02e-42e080c4d65b/metadata
2020-06-01T19:26:44Z [ERROR] [/server/error/error.go:53]: {"errors":[{"code":"INTERNAL_SERVER_ERROR","message":"scanner API: get metadata: scanner controller: ping: v1 client: get metadata: general response handler: invalid character '\u003c' looking for beginning of value"}]}
/harbor/src/server/error/error.go:81, github.com/goharbor/harbor/src/server/error.apiError
/harbor/src/server/error/error.go:50, github.com/goharbor/harbor/src/server/error.SendError
/harbor/src/common/api/base.go:67, github.com/goharbor/harbor/src/common/api.(*BaseAPI).RenderError
/harbor/src/common/api/base.go:228, github.com/goharbor/harbor/src/common/api.(*BaseAPI).SendInternalServerError
/harbor/src/core/api/scanners.go:71, github.com/goharbor/harbor/src/core/api.(*ScannerAPI).Metadata
/usr/local/go/src/reflect/value.go:460, reflect.Value.call
/usr/local/go/src/reflect/value.go:321, reflect.Value.Call
/harbor/src/vendor/github.com/astaxie/beego/router.go:853, github.com/astaxie/beego.(*ControllerRegister).ServeHTTP
/harbor/src/server/middleware/transaction/transaction.go:62, github.com/goharbor/harbor/src/server/middleware/transaction.Middleware.func1.1
/harbor/src/lib/orm/orm.go:63, github.com/goharbor/harbor/src/lib/orm.WithTransaction.func1
/harbor/src/server/middleware/transaction/transaction.go:71, github.com/goharbor/harbor/src/server/middleware/transaction.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/notification/notification.go:31, github.com/goharbor/harbor/src/server/middleware/notification.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/orm/orm.go:53, github.com/goharbor/harbor/src/server/middleware/orm.MiddlewareWithConfig.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/middleware.go:52, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/security/security.go:62, github.com/goharbor/harbor/src/server/middleware/security.Middleware.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/csrf/csrf.go:55, github.com/goharbor/harbor/src/server/middleware/csrf.attach.func1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/vendor/github.com/gorilla/csrf/csrf.go:297, github.com/gorilla/csrf.(*csrf).ServeHTTP
/harbor/src/server/middleware/csrf/csrf.go:75, github.com/goharbor/harbor/src/server/middleware/csrf.Middleware.func2
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/session/session.go:34, github.com/goharbor/harbor/src/server/middleware/session.Middleware.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/log/log.go:33, github.com/goharbor/harbor/src/server/middleware/log.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/harbor/src/server/middleware/requestid/requestid.go:37, github.com/goharbor/harbor/src/server/middleware/requestid.Middleware.func1
/harbor/src/server/middleware/middleware.go:57, github.com/goharbor/harbor/src/server/middleware.New.func1.1
/usr/local/go/src/net/http/server.go:2007, net/http.HandlerFunc.ServeHTTP
/usr/local/go/src/net/http/server.go:2802, net/http.serverHandler.ServeHTTP
/usr/local/go/src/net/http/server.go:1890, net/http.(*conn).serve
2020/06/01 19:26:44.930 [D] [transaction.go:62] | 10.254.207.28| 500 | 142.877256ms| match| GET /api/v2.0/scanners/aa21c639-a43d-11ea-b02e-42e080c4d65b/metadata r:/api/v2.0/scanners/:uuid/metadata
2020-06-01T19:26:47Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 7a4b3f47-8bac-487b-96eb-b053a6c700a1 to the logger for the request GET /api/v2.0/ping
2020-06-01T19:26:47Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="7a4b3f47-8bac-487b-96eb-b053a6c700a1"]: an unauthorized security context generated for request GET /api/v2.0/ping
2020/06/01 19:26:47.375 [D] [transaction.go:62] | 10.82.47.20| 200 | 323.212µs| match| GET /api/v2.0/ping r:/api/v2.0/ping
2020-06-01T19:26:57Z [DEBUG] [/server/middleware/log/log.go:30]: attach request id 632c89bd-d232-437b-9c68-091965887fbb to the logger for the request GET /api/v2.0/ping
2020-06-01T19:26:57Z [DEBUG] [/server/middleware/security/unauthorized.go:29][requestID="632c89bd-d232-437b-9c68-091965887fbb"]: an unauthorized security context generated for request GET /api/v2.0/ping
2020/06/01 19:26:57.375 [D] [transaction.go:62] | 10.82.47.20| 200 | 213.47µs| match| GET /api/v2.0/ping r:/api/v2.0/ping
```

Additionally, I found a couple of other issues with trivy.
1. It will try to create a persistent volume when persistence is set to false
2. It's not passing along the proxy values when adding trivy as one of the proxy components (since I'm running behind a proxy I had to add the proxy values manually to the template to test).

Thanks for your help!

**Comment by phisco** [comment]
same issue here 👍

**Comment by lfts** [comment]
Same for me

**Comment by heww** [comment]
Closed by #599
